tests/integration/suite/daprd/workflow/crossapp/basiclogline.go

```diff
@@ -0,0 +1,171 @@
+/*
```

Comment:
Using log matching to check for cross app functionality doesn't seem quite right.
We should test functionality by the fact of activities etc. being executed.